datafusion/sqllogictest/test_files/window.slt

```diff
@@ -3407,6 +3407,23 @@ WITH ORDER (a ASC, b ASC)
 WITH ORDER (c ASC)
 LOCATION '../core/tests/data/window_2.csv';
 
+query TT
+EXPLAIN SELECT *, SUM(a) OVER(ORDER BY b ASC, d ASC)
```

Comment:
Perhaps add a test like
`select * from table where b=0` as well? (sort should be removed)

Reply:
Added a new test, as in your suggestion. Thanks @Dandandan.